src/Components/ErrorsNotify/ErrorsNotify.tsx

```diff
@@ -0,0 +1,42 @@
+import { useContext, useEffect, useState } from 'react';
```

Comment:
no need to create a separate folder for each component if there are no subcomponents related only to the current component, own styles file, etc.

And in case a separate folder is necessary, it's a good practice to create index.ts file and re-export the component